phpBB/includes/ucp/ucp_pm_viewfolder.php

```diff
@@ -541,6 +533,7 @@ function get_pm_from($folder_id, $folder, $user_id)
 			USERS_TABLE			=> 'u',
 		),
 		'WHERE'		=> "t.user_id = $user_id
+		AND p.root_level = 0
```

Comment:
One more tab.

Also I think you reverted the query to original one again. Now you don't have correct (un)read status.

Reply:
This wasn't meant to be committed! Sorry about that